README.md

```diff
@@ -1,111 +1,175 @@
+For legal and compliance reasons, organizations or individuals can choose to provide an optional `licenseWhitelist` to compare against the [license of the repository](https://docs.github.com/en/rest/reference/licenses) being forked. If the license key returned by the Github API is not found within the provided whitelist, `forker` will exit without forking the repository.
 
-----
+## Inputs
 
-Use this template to bootstrap the creation of a TypeScript action.:rocket:
+### `token` (string, required)
 
-This template includes compilation support, tests, a validation workflow, publishing, and versioning guidance.  
+The Github API [token](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) you wish to use for automating fork creation. If you are using Github [encrypted secrets](https://docs.github.com/en/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow), you should reference the variable name you have defined for your secret.
 
-If you are new, there's also a simpler introduction.  See the [Hello World JavaScript Action](https://github.com/actions/hello-world-javascript-action)
+> 💡 **Note:** Ensure the token you are using has sufficient permissions to create repositories in your intended destination (either an organization or individual user account).
 
-## Create an action from this template
+**Example:** `${{ secrets.GH_API_TOKEN }}`
 
-Click the `Use this Template` and provide the new repo details for your action
+**Default:** `${{ github.token }}`
 
-## Code in Main
+### `owner` (string, required)
 
-> First, you'll need to have a reasonably modern version of `node` handy. This won't work with versions older than 9, for instance.
+The owner of the Github repository you wish to fork. Can be an organization or individual user account.
 
-Install the dependencies  
-```bash
-$ npm install
-```
+**Example:** `tremor-rs`
 
-Build the typescript and package it for distribution
-```bash
-$ npm run build && npm run package
+### `repo` (string, required)
+
+The name of the Github repository you wish to fork.
+
+**Example:** `tremor-runtime`
+
+### `org` (string, optional)
+
+The name of the destination Github organization where you wish to fork the specified repository.
+
+**Example:** `wayfair-contribs`
+
+### `user` (string, optional)
+
+The Github account for the person requesting the fork.
+
+> 💡 **Note:** This is only required if you are managing a Github organization, and wish to associate a specific user with the fork request. If neither `org` nor `user` inputs are specified, `forker` will default to forking the repository into your own Github account.
+
+**Example:** `lelia`
```

Comment:
What is the expected behaviour when `user` is a valid username and `org` is undefined. It would appear that `user` is simply ignored. Is that worth documenting? Perhaps capturing in a unit test?

Reply:
Ah, so when `user` is **not** used in concert with `org`, `forker` will default to creating a fork in the namespace of the user (or rather, the personal Github API token associated with a user) who is initiating the Action. 

You're right that it effectively becomes an ignored field in that circumstance, as I'm under the assumption that no individual Github user has the permissions to create a fork in _another_ Github user's namespace (as opposed to an org). We can definitely update the README to make that clearer.

Reply:
Updated phrasing to expand on the behavior when only `user` is passed